src/lib/components/ratio/ColorIssues.svelte

```diff
@@ -53,42 +41,27 @@
         When a color is out of gamut for the user's screen, the browser will
         adjust the color to be in gamut.
       </p>
-    </dd>
-  </dl>
-</details>
+    </li>
+  </ul>
+</div>
 
 <style lang="scss">
   @use 'config';
```

Comment:
Is this `@use 'config';` still needed?

```suggestion
```

Reply:
good catch. removed in https://github.com/oddbird/oddcontrast/pull/223/commits/39744edde75db38a9c8ede06bf432390db5a9acd